### PR TITLE
Add WMatic

### PR DESCRIPTION
**Pull request by JohnnyWyles**
https://app.osmosis.zone/pool/789
Requesting add to frontier
Assetlist pull request here: https://github.com/osmosis-labs/assetlists/pull/192

### CODE REVIEW

**Comment by sunnya97** [comment]
Is this Matic from Ethereum or from Polygon?

**Comment by JohnnyWyles** [comment]
> Is this $MATIC from Ethereum or from Polygon?

From Polygon, so WMATIC for their defi usage. Only one that Axelar has available to transfer.

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -1957,6 +1957,13 @@ chainInfos.push({
       coinGeckoId: "polkadot",
       coinImageUrl: "/tokens/dot.svg",
     },
+    {
+      coinDenom: "MATIC",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:1961]: this describes MATIC, while the token is still WMATIC. the symbol, minimal denom, cgid, and image name are not correct here.

Comment (JohnnyWyles) [reply packages/web/config/chain-infos.ts]: > this describes MATIC, while the token is still WMATIC. the symbol, minimal denom, cgid, and image name are not correct here.

Sorry, resolved.  Had it right in the other file, not sure what I was thinking.

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -1957,6 +1957,13 @@ chainInfos.push({
       coinGeckoId: "polkadot",
       coinImageUrl: "/tokens/dot.svg",
     },
+    {
+      coinDenom: "WMATIC",
+      coinMinimalDenom: "wmatic-wei",
+      coinDecimals: 18,
+      coinGeckoId: "wmatic",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:1964]: @sunnya97 is this where you want polygon's cgid being used?

**Comment by jonator** [comment]
@JeremyParish69 updates on this? Is this stale?

**Comment by jonator** [comment]
Can reopen if needed

**Pull request closed without merging**